easybuild/easyconfigs/l/libutempter/libutempter-1.1.6.2-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'libutempter'
+version = '1.1.6.2'
+
+homepage = 'http://git.altlinux.org/people/ldv/packages/?p=libutempter.git'
+description = """libutempter is library that provides an interface for terminal
+ emulators such as screen and xterm to record user sessions to utmp and wtmp files."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
+
+# Download from http://git.altlinux.org/people/ldv/packages/?p=libutempter.git;a=snapshot;h=1.1.6-alt2;sf=tgz
+# and unzip / rename libutempter-1.1.6-alt2-3c6e555.tar.gz to libutempter-1.1.6.2.tar
```

Comment:
@SimonPinches Maybe also mention that the unzipping is required to get a consistent checksum across systems?

And include the command to unzip & rename (although it's straightforward):

```python
# Download from http://git.altlinux.org/people/ldv/packages/?p=libutempter.git;a=snapshot;h=1.1.6-alt2;sf=tgz
# and unpack (required to obtain a consistent checksum) + rename to obtain libutempter-1.1.6.2.tar:
# gunzip libutempter-1.1.6-alt2-3c6e555.tar.gz && mv libutempter-1.1.6-alt2-3c6e555.tar libutempter-1.1.6.tar

Reply:
@boegel Updated as requested :-)